lib/environment.rb

```diff
@@ -450,7 +450,8 @@ def service_project
       unless @service_project
         # if the cluster set the default scheduler, set the project running debug pod node-selector=''
         # to overwrite the default scheduler, or the pod can not be run successfully
-        project_name = "prj-" + EXECUTOR_NAME.downcase
+        project_suffix = [*"a".."z", *"1".."9"]
```

Comment:
@jmekkatt Please add the comment here regarding why you are adding this step? so that other people can understand.